keps/sig-network/2681-pod-host-ip/README.md

```diff
@@ -284,11 +284,105 @@ The field is only informative, it doesn't affect running workloads.
 
 ###### What specific metrics should inform a rollback?
 
-The `status.hostIPs` field in Pod is empty, or frequently updated, or cause any other to crash.
+It will immediately update all running Pods on the node where this feature is enabled.
+
+If any of these phenomena imply that the feature is abnormal and needs to be rolled back, the `status.hostIPs` field in the pod is empty, or it is updated frequently, or it causes other Pods to crash.
```

Comment:
This is useful, but it isn't a metric.

I think that in particular for "updated frequently" we can actually use the apiserver_requests_total.
If it significantly increases for pods, that may suggest a problem, right?